fileicons/openhab-icon-theme.json

```diff
@@ -0,0 +1,150 @@
+{
+    "iconDefinitions": {
+        "_file": {
+            "iconPath": "./images/File_16x.svg"
+        },
+        "_folder": {
+            "iconPath": "./images/Folder_16x.svg"
+        },
+        "_folder-open": {
+            "iconPath": "./images/FolderOpen_16x.svg"
+        },
+        "_file_configure": {
+            "iconPath": "./images/File_16x_Configure.png"
```

Comment:
Looking at [vscode-icons](https://github.com/vscode-icons/vscode-icons) extension they seem to follow the 'SVG all the way' rule. I'm not sure where do they source their icons from. It might be  [material icons](https://material.io/icons/).
Let's go ahead with your PNGs now, and afterwards we can [create a new issue](https://github.com/openhab/openhab-vscode/issues/new) to change the icons to SVG utilizing some available iconset. Makes sense? 🙂